content/en/docs/getting-started/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Getting Started
+description: >-
+  Depending on your role, you might be looking for a different way to get 
+  started with OpenTelemetry. Find your way on how to get started with
+  OpenTelemetry.
+spelling: cSpell:ignore otel
+aliases: [/about, /docs/concepts/overview, /otel]
+weight: 1
+---
+
+Before you deep dive into your role specific _Getting Started_, you can try out
+our official [OpenTelemetry Demo](/docs) to **see** how observability with
+OpenTelemetry looks like.
+
+If none of the roles below is applicable to you, [let us know](https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Add%20a%20new%20persona:%20My%20Persona&body=Provide%20a%20description%20of%20your%20role%20and%20responsibilities%20and%20what%20your%20observability%20goals%20are)
+
+## Operations
+
+> I operate a set of applications in production.
+>
+> My goal is to get telemetry out of them without touching their code.
+>
+> I want to collect logs, metrics and traces from hundreds of services and.
```

Comment:
```suggestion
> I want to collect traces, metrics, and logs from several services and
```
Suggest ordering by maturity of signal and just saying "several" services, since OTel is relevant for any number of services